SUI/SUI/ContentView.swift

```diff
@@ -8,17 +8,113 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    private enum Constants {
+        static let firstText = "Пример Alert"
+        static let secondTestText = "foo"
+        static let secondText = "Пример Alert 2 с 2 кнопками и логикой"
+        static let firstAlertName = "Показать"
+        static let closeButtonText = "Close"
+        static let secondAlertName = "Показать"
+        static let secondAlertTitle = "Wanna change label?"
+        static let secondAlertPrimaryButtonText = "Change label"
+        static let firstActionSheetText = "Пример Action Sheet"
+        static let secondActionSheetText = "Пример Action Sheet с кнопками и логикой"
+        static let firstActionSheetName = "Показать"
+        static let actionSheetConfDialogText = "Are you sure?"
+        static let testButtonText = "test"
+        static let testPrint = "its worked"
+        static let secondActionSheetName = "Показать"
+        static let secondActionSheetButtonName = "Change color"
+    }
+    
+    @State var isFirstAlertShown = false
+    @State var isSecondAlertShown = false
+    @State var isActionButtonTapped = false
+    @State var isFirstActionSheetShown = false
+    @State var isSecondActionSheetShown = false
+    @State var isSecondActionSheetButtonTapped = false
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        VStack(spacing: 15) {
+            Text(Constants.firstText)
+            showFirstAlert()
+            if !isActionButtonTapped { Text(Constants.secondText) } else {
```

Comment:
```suggestion
            if !isActionButtonTapped { 
            Text(Constants.secondText)
             } else {
```
Тут и дальше много пишешь в одну строчку.